Trappist/src/Promact.Trappist.Web/Startup.cs

```diff
@@ -51,28 +65,38 @@ public void ConfigureServices(IServiceCollection services)
                 .AddEntityFrameworkStores<TrappistDbContext>()
                 .AddDefaultTokenProviders();
 
-            services.AddMvc();
-
-            services.AddScoped<IQuestionsRespository, QuestionsRepository>();
-        }
+            services.AddMvc(/*config => { config.Filters.Add(typeof(GlobalExceptionFilter)); }*/)
+                 .AddJsonOptions(o => o.SerializerSettings.NullValueHandling = Newtonsoft.Json.NullValueHandling.Ignore)
+           .AddJsonOptions(o => o.SerializerSettings.ContractResolver = new DefaultContractResolver());
+            ;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this empty statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1116)